### ISSUE TITLE
[Feature Request] Install to path instead of its own domain

### ISSUE THREAD

**Issue by minnixtx** [YunoHost-Apps/2FAuth_ynh#22]
For the next iteration of this application, would it be possible to set it up to install to the yunohost domain on path instead of its own domain? For example www.myyunoserver.com/2fauth

**Comment by ericgaspar** [comment]
I'm not sure if this is allowed upstream...

**Comment by Bubka** [comment]
Indeed, This is not supported at this time by 2FAuth. But it will be possible with the next major version (4.0)

**Comment by artlog** [comment]
As of today upstream version used is 6.1.3. This is then no more an upstream issue.
